src/ints/bios_disk.cpp

```diff
@@ -414,7 +424,23 @@ static Bitu INT13_DiskHandler(void) {
 		bufptr = reg_bx;
 		for (Bitu i = 0; i < reg_al; i++) {
 			last_status = imageDiskList[drivenum]->Read_Sector((uint32_t)reg_dh, (uint32_t)(reg_ch | ((reg_cl & 0xc0)<< 2)), (uint32_t)((reg_cl & 63)+i), sectbuf);
-			if((last_status != 0x00) || (killRead)) {
+
+			// Determine delay based on whether it's a floppy or
+			// hard disk
+			if (imageDiskListTypes[drivenum] == DiskType::Floppy) {
+				diskio_delay(512, floppy_noise.get(), DiskType::Floppy);
```

Comment:
What is the magic value of 512? Why 512? Please extract it to a constant with an explanation.